src/Conversions.cc

```diff
@@ -274,11 +274,11 @@ msgs::Material ignition::gazebo::convert(const sdf::Material &_in)
   out.set_lighting(_in.Lighting());
   out.set_double_sided(_in.DoubleSided());
 
-  sdf::Pbr *pbr = _in.PbrMaterial();
+  auto pbr = _in.PbrMaterial();
   if (pbr)
   {
-    msgs::Material::PBR *pbrMsg = out.mutable_pbr();
-    sdf::PbrWorkflow *workflow = pbr->Workflow(sdf::PbrWorkflowType::METAL);
+    auto pbrMsg = out.mutable_pbr();
```

Comment:
This type's const-ness was already correct.

Reply:
Oh you're right. I'm ok letting this through to save another round of CI.